test.sh

```diff
@@ -39,25 +46,24 @@ do
             if ! [[ -z $full_out ]] ; then
                 sleep $output_time
                 echo -e "\e[1;31m build failing \e[1;39m: $code"
```

Comment:
Console output coloring somewhy doesn't work in macOS. We might either fix it or turn it off there, such that '\e[1;32m' wouldn't show up in Travis output.

Reply:
Not that it is related to cmake changeset, but I fixed it. See newest travis output.